package.json

```diff
@@ -37,6 +37,7 @@
     ]
   },
   "dependencies": {
+    "@ember-template-lint/todo-utils": "^3.1.0",
```

Comment:
We should probably update this to latest (3.2.0)

Reply:
Yep, I just published that yesterday so that seems fine. It's only got the due days functionality in it.

Reply:
Done.